### PR TITLE
CSOAR-362 - Cloud SOAR APIs

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds the following new article:
https://help.sumologic.com/docs/cloud-soar/cloud-soar-apis

Issue number: 
[CSOAR-362](https://jira.kumoroku.com/jira/browse/CSOAR-362)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/cloud-soar/mssp.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 id: mssp
-title: Cloud SOAR for MSSPs
+title: MSSP
```
Comment (kimsauce) [inline docs/cloud-soar/mssp.md:3]: For SEO purposes, I would do:

`title` Cloud SOAR for MSSPs
`sidebar_label` MSSPs

File: docs/cloud-soar/cloud-soar-apis.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+| Deployment | Documentation URL                                         |
+|:-----------|:----------------------------------------------------------|
+| AU         | https://api.au.sumologic.com/docs/csoar/  |
+| CA         | https://api.ca.sumologic.com/docs/csoar/  |
+| DE         | https://api.de.sumologic.com/docs/csoar/  |
+| EU         | https://api.eu.sumologic.com/docs/csoar/  |
+| IN         | https://api.in.sumologic.com/docs/csoar/  |
+| JP         | https://api.jp.sumologic.com/docs/csoar/  |
+| US1        | https://api.sumologic.com/docs/csoar/     |
+| US2        | https://api.us2.sumologic.com/docs/csoar/ |
```
Comment (kimsauce) [inline docs/cloud-soar/cloud-soar-apis.md:21]: Will you copy this over to [Sumo APIs](https://help.sumologic.com/docs/api/)?

**Review by kimsauce — CHANGES_REQUESTED** [review]
a couple comments, otherwise looks great

**Comment by jpipkin1** [comment]
@kimsauce, thanks for the review. That's an awesome catch about copying over the Cloud SOAR APIs article to [Sumo APIs](https://help.sumologic.com/docs/api/)!

File: docs/cloud-soar/mssp.md (hunk 1 above)
Comment (jpipkin1) [reply docs/cloud-soar/mssp.md]: Yep! I had them backwards.

File: docs/cloud-soar/cloud-soar-apis.md (hunk 2 above)
Comment (jpipkin1) [reply docs/cloud-soar/cloud-soar-apis.md]: Done.

**Comment by jpipkin1** [comment]
@kimsauce, could you take another look?

**Review by kimsauce — APPROVED** [review]
Thank you, @jpipkin1 - looks great!

**Comment by jpipkin1** [comment]
Thanks, @kimsauce!

**Pull request merged**